src/accessibility/describe.js

```diff
@@ -346,99 +327,88 @@ function _elementName(name) {
   return name;
 }
 
-//creates fallback HTML structure for element descriptions
-function _descElementFallbackHTML(dummyDOM, cnvId, name) {
-  //if there is no description container
-  if (!dummyDOM.querySelector(`#${cnvId + descContainer}`)) {
-    //if there are no accessible outputs (see textOutput() and gridOutput())
-    if (!dummyDOM.querySelector(`#${cnvId}accessibleOutput`)) {
-      //create container + table for element descriptions
-      dummyDOM.querySelector('#' + cnvId).innerHTML = `<div id="${cnvId +
-        descContainer}" role="region" aria-label="Canvas Description"><table id="${cnvId +
-        fallbackTableId}"><caption>Canvas elements and their descriptions</caption></table></div>`;
-    } else {
-      //create container + table for element descriptions before outputs
-      dummyDOM
-        .querySelector(`#${cnvId}accessibleOutput`)
+//creates HTML structure for element descriptions
+p5.prototype._describeElementHTML = function(type, name, text) {
+  const cnvId = this.canvas.id;
+  if (type === 'fallback') {
+    //if there is no description container
+    if (!this.dummyDOM.querySelector(`#${cnvId + descContainer}`)) {
+      //if there are no accessible outputs (see textOutput() and gridOutput())
+      let html = `<div id="${cnvId}${descContainer}" role="region" aria-label="Canvas Description"><table id="${cnvId}${fallbackTableId}"><caption>Canvas elements and their descriptions</caption></table></div>`;
+      if (!this.dummyDOM.querySelector(`#${cnvId}accessibleOutput`)) {
+        //create container + table for element descriptions
+        this.dummyDOM.querySelector('#' + cnvId).innerHTML = html;
+      } else {
+        //create container + table for element descriptions before outputs
+        this.dummyDOM
+          .querySelector(`#${cnvId}accessibleOutput`)
+          .insertAdjacentHTML('beforebegin', html);
+      }
+      //if describe() has already created the container and added a description
```

Comment:
at first, I didn't read this as going with the `else` below because the indentation matched above

Reply:
umm that's a good point, that's the way lint makes it look. I'll move the comment after the else!